lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -2081,7 +2081,9 @@ export class StandardController implements IController {
                             const isSilentlyResolvable =
                                 (!(
                                     refreshTokenError instanceof
-                                    InteractionRequiredAuthError
+                                        InteractionRequiredAuthError &&
+                                    refreshTokenError.subError !==
```

Comment:
This logic is really hard to follow. It may be time to factor this into a helper function with each condition separated to make it more readable with comments and such